src/main/java/stirling/software/SPDF/SPdfApplication.java

```diff
@@ -64,7 +64,8 @@ public static void main(String[] args) throws IOException, InterruptedException
         try {
             Thread.sleep(1000);
         } catch (InterruptedException e) {
-            throw new InterruptedException("Error");
```

Comment:
I think we should just re-throw it as a runtime exception

Reply:
done